scripts/mkcloud

```diff
@@ -128,6 +128,9 @@ scp="scp $sshopts"
 : ${install_chef_suse_override:=./install-chef-suse.sh}
 : ${cct_tests:="features:base"}
 
+# Replace special 'jumbo' with numeric so we can compare it later on
+[[ ${want_mtu_size} == jumbo ]] && want_mtu_size=9000
```

Comment:
I think, we already dropped the use of the "jumbo" value - but didnt update docs.
And if we wanted to add such a line, it should go into qa_crowbarsetup which is sourced here, so also active on the host.